Description: Move Pyro4 dependency
 It is required by the server component, but it wasn't added to "setup.py",
 only to "tox.ini". This way it wasn't being detected as a dependency of the
 binary package.
Forwarded: https://github.com/andsens/bootstrap-vz/commit/9e332b2
Author: Tiago Ilieve <tiago.myhro@gmail.com>

--- a/setup.py
+++ b/setup.py
@@ -27,6 +27,7 @@
                         'docopt >= 0.6.1',
                         'pyrfc3339 >= 1.0',
                         'requests >= 2.4.3',
+                        'pyro4 >= 4.30',
                         ],
       license='Apache License, Version 2.0',
       description='Bootstrap Debian images for virtualized environments',
--- a/tox.ini
+++ b/tox.ini
@@ -26,7 +26,6 @@
     nose
     nose-cov
     nose-htmloutput
-    Pyro4 >= 4.30
     pyvbox >= 0.2.0
 commands = nosetests --with-html --html-file=system.html --verbose {posargs:tests/system}
 
